client/src/components/settings/toggle-setting.css

```
@@ -78,52 +82,49 @@
   cursor: default;
 }
 
-.toggle-radio-group {
-  display: flex;
-  align-items: start;
-}
-
-.toggle-radio-group label {
-  font-weight: normal;
-  display: flex;
-  align-items: center;
+.toggle-radio-group label + label {
```

```suggestion
.toggle-radio-group label {
  display: inline-flex;
  align-items: center;
}

.toggle-radio-group label + label {
```

Using flexbox on the label will ensure that the radio buttons is always lined up properly with the text no matter what font size the user is viewing the page with.

---

Changed it! 
I remember when I first thought to implement input:checked, there came a problem which I forgot up until now. 
It was, we cannot add cursor to label when the input is checked. As label is the parent of input, I am unable to make that change.